pkg/vault/package.json

```diff
@@ -26,7 +26,7 @@
     "prettier": "npx prettier --write --plugin=prettier-plugin-solidity 'contracts/**/*.sol' 'test/**/*.sol'",
     "test": "yarn test:hardhat && yarn test:forge",
     "test:hardhat": "hardhat test",
-    "test:forge": "yarn build && ./forge-test.sh",
+    "test:forge": "yarn build && REUSING_HARDHAT_ARTIFACTS=true ./forge-test.sh",
```

Comment:
Just because it makes the vault's `test:forge` similar to other packages `test:forge`, it doesn't change anything, but it's also easier to delete the argument if we want to run the tests with forge.